spec/facility_spec.rb

```diff
@@ -23,4 +32,76 @@
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
   end
+
+  #facility_1 & facility_2
+  it 'can register vehicles' do
+    expect(facility_1.add_service('Vehicle Registration')).to eq(["Vehicle Registration"])
```

Comment:
This could probably just be a command query
```ruby
facility_1.add_service('Vehicle Registration')
```
then have an expect statement like
```ruby
expect(facility_1.services).to eq(['Vehicle Registration'])
```